### PR TITLE
feat: Backport of RES interceptor

### PR DESCRIPTION
**Pull request by johanandren**
Backport of #32526 from main

### CODE REVIEW

**Review by patriknw — APPROVED** [review]
LGTM

File: akka-persistence-typed/src/main/mima-filters/2.9.5.backwards.excludes/32526-replicated-interceptor.excludes [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# new method
```
Comment (patriknw) [inline akka-persistence-typed/src/main/mima-filters/2.9.5.backwards.excludes/32526-replicated-interceptor.excludes:1]: always confusing when starting to branch off, maybe this should be 2.9.6 and a copy in 2.8.x ?

Comment (johanandren) [reply akka-persistence-typed/src/main/mima-filters/2.9.5.backwards.excludes/32526-replicated-interceptor.excludes]: Just pushed, didn't think I needed to put it elsewhere than 2.9.6. Why is the copy needed in a release branch?

**Comment by johanandren** [comment]
ghactions are very slow today

**Pull request merged**